packages/mdc-ripple/foundation.js

```diff
@@ -505,32 +504,27 @@ class MDCRippleFoundation extends MDCFoundation {
     const state = /** @type {!ActivationStateType} */ (Object.assign({}, activationState));
 
     if (activationState.isProgrammatic) {
-      const evtObject = null;
-      requestAnimationFrame(() => this.animateDeactivation_(evtObject, state));
+      requestAnimationFrame(() => this.animateDeactivation_(state));
       this.resetActivationState_();
     } else {
       this.deregisterDeactivationHandlers_();
       requestAnimationFrame(() => {
         this.activationState_.hasDeactivationUXRun = true;
-        this.animateDeactivation_(e, state);
+        this.animateDeactivation_(state);
         this.resetActivationState_();
       });
     }
   }
 
-  /**
-   * @param {?Event=} event Optional event containing position information.
-   */
-  deactivate(event = null) {
-    this.deactivate_(event);
+  deactivate() {
+    this.deactivate_();
   }
 
   /**
-   * @param {Event} e
    * @param {!ActivationStateType} options
    * @private
    */
-  animateDeactivation_(e, {wasActivatedByPointer, wasElementMadeActive}) {
+  animateDeactivation_({wasActivatedByPointer, wasElementMadeActive}) {
```

Comment:
Nice catch on the event not actually being used in the whole deactivate flow... I must've missed that as I was gradually refactoring stuff out.